modules/dreamview/frontend/src/styles/main.scss

```diff
@@ -661,6 +661,94 @@ body {
         }
     }
 
+    .show-electricity{
+        position: absolute;
+        width: 224px;
+        height: 35px;
+        top: 287px;
+        right: 20px;
+
+        font-size: 14px;
+
+        .battery-icon {
+            position: absolute;
+            width: 116px;
+            height: 35px;
+            background: rgba(0, 0, 0, 0.5);
+        }
+
+        .electricity-percentage {
+            position: absolute;
+            top: 0px;
+            right: 0px;
+            width: 105px;
+            height: 35px;
+            background: rgba(0, 0, 0, 0.5);
+
+            .text {
```

Comment:
This .text style is repeated in .gears-label.label and current-gears.text. Please try reuse the same code.  One possible approach is to use SCSS's extend feature, or simply define a common one for React component to refer to.